settings/enginx_jaws/jaws.cmd

```diff
@@ -0,0 +1,8 @@
+############ Enginx jaws ##############

```

Comment:
Should this be in the PR?  Seems strange an ENGINX file is here...

Reply:
Whilst doing the ticket I noticed that there was no example of how to do the Enginx jaws so thought i'd add one quickly. I probably should have done it in a separate branch, sorry.

Reply:
Fair enough - was just a bit confusing.